.gitignore

```diff
@@ -2,6 +2,7 @@
 .DS_Store
 .vscode
 morgan.log
+package-lock.json
```

Comment:
same as the other issue, this shoudn't be ignored

Reply:
Fixed